Makefile

```diff
@@ -3,6 +3,14 @@ fmt:
 		clang-format -i --sort-includes $${file} ; \
 	done
 
+check-fmt:
+	for file in ./src/**/*.[ch] ; do \
+		var=`clang-format --dry-run -Werror $${file}` ; \
+		if [ $$var -ne 0 ] ; then \
```

Comment:
Currently, the `ini.c` and `ini.h` are not formatted - so this should fail. In the action logs there is 
```
/bin/sh: line 3: [: -ne: unary operator expected
```
Not sure how to resolve this.